# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/setserial/lx_types_h.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./setserial.c.orig	Sun Nov 14 12:29:08 1999
+++ ./setserial.c	Sun Nov 14 12:30:09 1999
@@ -22,6 +22,7 @@
 #ifdef HAVE_LINUX_HAYESESP_H
 #include <linux/hayesesp.h>
 #endif
+#include <linux/types.h>
 #include <linux/serial.h>
 
 #define VERSION_STR "2.15"
